examples/client/lwm2mclient.c

```diff
@@ -1014,6 +1015,22 @@ int main(int argc, char *argv[])
         case '4':
             data.addressFamily = AF_INET;
             break;
+        case 's':
```

Comment:
Duplicate case if WITH_TINYDTLS is defined.

Reply:
@rettichschnidi, the CI didn't catch this. Possibly another area for improvement. I'm not sure if it is doing a DTLS build or not.